subsys/bluetooth/shell/gatt.c

```diff
@@ -236,6 +236,7 @@ int cmd_gatt_read(int argc, char *argv[])
 	read_params.handle_count = 1;
 	read_params.single.handle = strtoul(argv[1], NULL, 16);
 
+	read_params.single.offset = 0;
```

Comment:
Fixed. Originally, it tried to use logical groupping keeping statements dealing with single.offset together.